addons/optionsmenu/functions/fnc_onListBoxSettingsChanged.sqf

```diff
@@ -16,26 +16,30 @@
 
 #include "script_component.hpp"
 
-private ["_settingIndex", "_rightDropDownIndex"];
+private ["_settingsMenu", "_settingIndex", "_rightDropDownIndex"];
 
-_settingIndex =  lbCurSel 200;  //Index of left list
 _rightDropDownIndex = lbCurSel 400;  //Index of right drop down
-
 if (_rightDropDownIndex < 0) then {_rightDropDownIndex = 0;};
 
+_settingIndex = -1;
+if (((lbCurSel 200) >= 0) && {(lbCurSel 200) < ((lbSize 200)/2)}) then {
```

Comment:
We were using listBox commands on a listBoxN.
It worked, but probably safer and easier to read to swap to the proper commands.